doc/api/errors.md

```diff
@@ -2984,9 +2984,11 @@ import 'package-name'; // supported
 
 ### `ERR_UNSUPPORTED_RESOLVE_REQUEST`
 
-Calling `import.meta.resolve` with a bare specifier outside of a `file:` module
-is unsupported. Calling `import.meta.resolve` with a relative URL from a `data:`
-module is unsupported.
+Calling `import.meta.resolve` with a [relative URL][] is only allowed as defined
+by the spec, and the URL will be resolved relative to the URL of the module.
+
+Calling `import.meta.resolve` with a bare specifier is allowed only for builtin
+modules, or from a module whose URL is using `file` scheme.
```

Comment:
```suggestion
Calling `import.meta.resolve` with a bare specifier is only allowed from
modules loaded from the filesystem.
```

It doesn’t make much sense to call `import.meta.resolve('node:fs')`, I don’t think we need to cover that case, just as you’re excluding the case of resolving absolute URLs.